servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@calvinjia Sorry for that I have made a mistake in the `getDirCandidate()` function in `PartialLRUEvictor.java`. The blocks should be evicted by LRU only if it exists in the selected `StorageDir`, but I evict in the whole `BlockStoreLocation`. However,  I have to scan the whole blocks to decide the LRU blocks to evict in the selected `StorageDir` or should I matain LRU blocks in each `StorageDir`?